sonar-project.properties

```diff
@@ -7,7 +7,7 @@ sonar.coverage.exclusions=\
   **/jest.config.ts,scripts/*,\
   packages/common/testing/*,
   eslint.config.mjs, \
-  **/tests/*
+  **/tests/**/*
```

Comment:
This change is to exclude all files in the `tests` directories recursively. I still think the old one should have excluded the file `packages/getMyPrescriptions/tests/toMatchJsonLogMessage.ts`, but it didn't. This one does.